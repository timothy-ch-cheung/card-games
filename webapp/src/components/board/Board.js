import React from "react";
import Tile from "../tile/Tile";

class Board extends React.Component {

    render() {
        return (
            <div className="container">
                <style type="text/css">
                    {`
                        .board {
                            display: grid;
                            grid-gap: 1px;
                            grid-template-columns: repeat(3, 1fr);
                            width: 300px;
                            height: 300px;
                            grid-gap: 0;
                            margin: 20px auto;
                        }    
                    `}
                </style>
                <div className="board">
                    <Tile/><Tile/><Tile/>
                    <Tile/><Tile/><Tile/>
                    <Tile/><Tile/><Tile/>
                </div>
            </div>
        )
    }
}

export default Board;