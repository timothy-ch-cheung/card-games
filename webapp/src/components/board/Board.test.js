import React from 'react';
import renderer from "react-test-renderer";
import Board from "./Board";

test('matches snapshot', () => {
    const tree = renderer.create(<Board/>).toJSON();
    expect(tree).toMatchSnapshot()
});