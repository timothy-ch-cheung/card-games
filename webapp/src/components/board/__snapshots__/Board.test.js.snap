// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="container"
>
  <style
    type="text/css"
  >
    
                        .board {
                            display: grid;
                            grid-gap: 1px;
                            grid-template-columns: repeat(3, 1fr);
                            width: 300px;
                            height: 300px;
                            grid-gap: 0;
                            margin: 20px auto;
                        }    
                    
  </style>
  <div
    className="board"
  >
    <button
      className="sc-AxjAm cYkxHS btn-tile"
      data-testid="tile"
    />
    <button
      className="sc-AxjAm cYkxHS btn-tile"
      data-testid="tile"
    />
    <button
      className="sc-AxjAm cYkxHS btn-tile"
      data-testid="tile"
    />
    <button
      className="sc-AxjAm cYkxHS btn-tile"
      data-testid="tile"
    />
    <button
      className="sc-AxjAm cYkxHS btn-tile"
      data-testid="tile"
    />
    <button
      className="sc-AxjAm cYkxHS btn-tile"
      data-testid="tile"
    />
    <button
      className="sc-AxjAm cYkxHS btn-tile"
      data-testid="tile"
    />
    <button
      className="sc-AxjAm cYkxHS btn-tile"
      data-testid="tile"
    />
    <button
      className="sc-AxjAm cYkxHS btn-tile"
      data-testid="tile"
    />
  </div>
</div>
`;
